run_kytos.py

```diff
@@ -0,0 +1,6 @@
+import cmd
```

Comment:
@Alopalao, looks like this file that you use locally got accidentally committed, could you double check?

Reply:
Yes, sorry.